commandLine/src/projects/CBLinuxProject.h

```diff
@@ -13,9 +13,9 @@ class CBLinuxProject: public CBWinProject {
 public:
 	CBLinuxProject(const std::string & target) : CBWinProject(target) {};
 
-	bool createProjectFile();
-	void addInclude(const fs::path & includeName){};
-	void addLibrary(const LibraryBinary & lib){};
+	virtual bool createProjectFile() override;
+	virtual void addInclude(const fs::path & includeName) override{};
+	virtual void addLibrary(const LibraryBinary & lib)override{};
```

Comment:
we can remove virtual here once it is final

Reply:
I just updatetd it. THanks for pointing out. Although, the linux test fails immediately when trying to pull ubuntu

Reply:
Thank you @roymacdonald let's merge it.